README-CAPROVER.md

```diff
@@ -0,0 +1,136 @@
+# Congratulations
+
+You know have a running foswiki and solr container, and need to finalize some settings to ensure your work is complete.
+
+## Necessary CapRover Changes
+
+* Enable Force HTTPS.
+* Enable HTTPS.
+
+## Docker Administration
+
+<a name="ContainerLogin"></a>
+
+### Container Login
+
+Login to the docker machine by identifying the container `docker ps` .
+
+```shell
+root@ubuntu-8gb-nbg1-1:~# docker ps
+CONTAINER ID        IMAGE                               COMMAND                  CREATED             STATUS              PORTS                                      NAMES
+d392b0392d5e        timlegge/docker-foswiki:latest      "sh docker-entrypoin…"   13 hours ago        Up 13 hours         80/tcp                                     srv-captain--foswiki.1.rl7vmf0qep9e8lbp2hqi4hp9o
+```
+
+and then using `docker exec -ti d392b0392d5e /bin/bash` to login to the machine.
+
+### Set the Admin Password via the Container
+
+1. `cd /var/www/foswiki/`
+1. `tools/configure -save -set {Password}='MyPassword'`
+
+<a name="CapRoverAdmin"></a>
+
+## CapRover Administration
+
+### Setting up HTTPS
+
+1. Login to CapRover Dashboard.
+1. Select the app created.
+1. Choose Enable HTTPS.
+1. Tick the checkbox "Force HTTPS by redirecting all HTTP traffic to HTTPS"
+1. Click on `Save & Update`.
+
+<a name="FoswikiAdmin"></a>
+
+## Foswiki Administration Login
+
+1. Connect to <https://foswiki.exapmle.com/>
+1. Login as the admin user (username: admin, password: [as set above])
+1. Access <https://foswiki.exapmle.com/bin/configure>
+
+### Start Modifications
+
+These settings are needed to ensure that your application can load java scripts and images.
+
+1. Login to [foswiki](#FoswikiAdmin) as an administrator.
```

Comment:
Also need to ensure the default URL and the allowed URLs I guess is set to the https url